### ISSUE TITLE
Store and expose the branch and git remote as part of our builds JSON 

### ISSUE THREAD

**Issue by KrauseFx** [fastlane/ci#481]
<img width="775" alt="screenshot 2018-04-02 02 36 34" src="https://user-images.githubusercontent.com/869950/38186306-ed05cbd8-3684-11e8-9418-441f3b2057ba.png">

For the build list we want to also show the branch information, as the sha isn't useful for the user, but a branch name (e.g. `add-about-screen`) is useful.

**More context**

<img width="481" alt="screenshot 2018-04-02 02 49 09" src="https://user-images.githubusercontent.com/869950/38186323-04b3a340-3685-11e8-8081-8601e8f1ed32.png">

**Comment by KrauseFx** [comment]
Working on this now 🚀

**Comment by KrauseFx** [comment]
Built with https://github.com/fastlane/ci/pull/809

**Comment by nakhbari** [comment]
I think `master` vs a sha is more comprehensible. Do you think we might want to do a combination since there can be different points in master? Something like `master@032cbjl`. Or a new column in the table?

**Comment by taquitos** [comment]
@nakhbari I think that's a great idea. I often search by SHA to know which build I need to kick off (in case of a specific failed build)

**Comment by KrauseFx** [comment]
yeah that's a great idea, I'll go ahead and create a new issue for this

**Comment by KrauseFx** [comment]
I personally never went to a CI system, selected a project, and then searched for a sha, as I always land on the CI system by clicking the `Details` button on GitHub instead 👍

**Comment by KrauseFx** [comment]
@nakhbari here we go https://github.com/fastlane/ci/issues/815
